_data/organizers/organizing_committee.yml

```diff
@@ -0,0 +1,164 @@
+- name: "Thomas Caswell"
```

Comment:
Can we sort this and the chairs section by alphabetical order (last name first) plx?

Reply:
Sure. To clarify, should I just sort by alphabetical order of last names first or both sort by last name and display the name in the format ("last name" "first name")?

Reply:
Done!